scripts/Jenkinsfile_batch_install_test

```diff
@@ -88,7 +88,7 @@ node {
                   break             
                 case "restartservice":
                   jobs[test_name] = { build job: "${env.PRODUCT_NAME}_restart_service", parameters: install_params }
-                  jobs[test_name] = { build job: "${env.PRODUCT_NAME}_restart_service_splitroles", parameters: install_params }
+                  jobs["restartservice_splitroles"] = { build job: "${env.PRODUCT_NAME}_restart_service_splitroles", parameters: install_params }
```

Comment:
nit: `jobs["restartservice_splitroles"]` to `jobs[test_name+"_splitroles"]` ?

Reply:
fixed